.ci/Jenkinsfile-compile

```diff
@@ -89,7 +89,7 @@ pipeline {
           def nuttx_builds_archive = [
             target: ["px4fmu-v2_default", "px4fmu-v3_default", "px4fmu-v4_default", "px4fmu-v4pro_default", "px4fmu-v5_default", "px4fmu-v5_rtps", "px4fmu-v5_stackcheck",
                      "aerofc-v1_default", "aerocore2_default", "auav-x21_default", "av-x-v1_default", "crazyflie_default", "mindpx-v2_default",
-                     "nxphlite-v3_default", "tap-v1_default", "omnibus-f4sd_default"],
+                     "nxphlite-v3_default", "omnibus-f4sd_default"],
```

Comment:
I had thought that this line would pass the Jenkins CI pipeline with the deprecated files, although perhaps it does not reference the `.ci/` directory changes for running the tests.  Please let me know if I have overlooked anything in this regard!